src/features/documentReferences/DocumentReferenceCard.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useMemo } from "react";
+
+import type { IDocumentReference } from "@ahryman40k/ts-fhir-types/lib/R4";
+import CalendarTodayIcon from "@mui/icons-material/CalendarToday";
+import { Card, CardContent, CardHeader, Typography } from "@mui/material";
+import { makeStyles } from "@mui/styles";
+import { DateTime } from "luxon";
+import { useTranslation } from "react-i18next";
+
+import Tag from "common/components/Tag";
+import ResourceCardActions from "features/resources/ResourceCardActions";
+
+import { TERMINOLOGY_SYSTEM_URL } from "../../constants";
+
+const useStyles = makeStyles((theme) => ({
+  flexContainer: {
+    display: "flex",
+    flexDirection: "row",
```

Comment:
"row" is the default value to "flexDirection", usually you don't have to specify